app/android/app/src/main/kotlin/com/example/my_project/NotificationOnKillService.kt

```diff
@@ -20,18 +20,23 @@ class NotificationOnKillService: Service() {
     private lateinit var description: String
 
     override fun onStartCommand(intent: Intent?, flags: Int, startId: Int): Int {
-        title = intent?.getStringExtra("title") ?: "Default title"
-        description = intent?.getStringExtra("description") ?: "Default body"
+        title = intent?.getStringExtra("title") ?: ""
+        description = intent?.getStringExtra("description") ?: ""
 
         return START_STICKY
     }
 
     @RequiresApi(Build.VERSION_CODES.O)
     override fun onTaskRemoved(rootIntent: Intent?) {
         try {
+            
+            if (title.isBlank() || description.isBlank()) {
```

Comment:
Can you explain this changes?

Reply:
Yes, I looked into this issue. This should fix it.

https://discord.com/channels/1192313062041067520/1231903583717425153/1272948353667432448